diagnostic.md

```diff
@@ -58,13 +61,21 @@ Suppose you want to define a mixin named `row` stored in `./row.scss`. Write the
 code to import the mixin definition in the current module.
 
 ```scss
-// your response here
+@import 'row'
 ```
 
 Now that the mixin is imported, let's use it. This mixin doesn't take any
 arguments. Write the code to include the mixin inside all elements with a
 class of `content`.
 
 ```scss
-// your response here
+@mixin row() {
+  @if map-has-key() {
+        @media #{inspect(map-get())} {
+      @content;
+    }
+  } @else {
+    @warn `#{}`.";
+  }
+}
 ```
```

Comment:
I think you overthought this one. The idea was to assume `@mixin row` was already defined in `row.scss`. So to use it, we'd call:

```CSS
.content {
  @include row;
}
```